Library/Formula/node.rb

```diff
@@ -2,9 +2,9 @@
 class Node < Formula
   desc "Platform built on Chrome's JavaScript runtime to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v0.12.7/node-v0.12.7.tar.gz"
-  sha256 "b23d64df051c9c969b0c583f802d5d71de342e53067127a5061415be7e12f39d"
-  head "https://github.com/nodejs/node.git", :branch => "v0.12"
+  url "https://nodejs.org/dist/v4.0.0/node-v4.0.0.tar.gz"
+  sha256 "e110e5a066f3a6fe565ede7dd66f3727384b9b5c5fbf46f8db723d726e2f5900"
+  head "https://github.com/nodejs/node.git", :branch => "v4.x"
```

Comment:
The [4.0.0 release notes](https://nodejs.org/en/blog/release/v4.0.0/) seem to indicate that future development will be on a (yet to be created) v5.x branch, and that the v4.x branch will be sort of LTS/"patches only", in line with their move to semantic versioning.